nemo/collections/nlp/modules/common/megatron/megatron_decoders.py

```diff
@@ -88,6 +88,8 @@ def get_decoder_model(
     moe_dropout=0.0,
     turn_off_rop=False,  # turn off the RoP positional embedding
     version=1,
+    position_embedding_type='learned_absolute',
+    use_flash_attention=False,
```

Comment:
if you're going to add this, shouldn't you add it to the t5 configs as well?

Reply:
Added